users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -49,7 +49,8 @@ def add_specaug_layer(
                 freq_mask_max_proportion (float): The maximum proportion of the frequency axis that can be masked. Default is 1.
                 max_time_num_seq_len_divisor (int):
                     The divisor for the sequence length to determine the maximum number of time masks. Default is 0.7.
-        num_epochs (int, optional): The total number of epochs for which the training will run. default 450
+                enable_logging (bool): Whether to enable detailed logging via print statements. Defaults to False.
+        num_epochs (int, optional): The total number of epochs for which the training will run. default 450.
```

Comment:
Why is the indentation inconsistent with the lines above?

Reply:
Is it? It looks fine in my editor. it should be in line with the rest of the dict entries.

Reply:
Please have a look here in the PR in the browser. Maybe it's about spaces vs. tabs?